mcs/class/System.IO.Compression/SharpCompress/Archive/Zip/ZipWritableArchiveEntry.cs

```diff
@@ -46,11 +45,6 @@ public override long Size
             get { return size; }
         }
 
-        public override DateTime? LastModifiedTime
-        {
-            get { return lastModified; }
-        }
```

Comment:
I don't really get why we're removing this? What about the other properties like `Size`, should they use the base implementation too?